Gemfile

```diff
@@ -3,3 +3,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
+  gem 'coveralls', '~> 0.8.2'
```

Comment:
Was just following this https://github.com/backus/devtools/blob/feature/flay-config-exclude-files/spec/spec_helper.rb#L7-L12